docs/File-Formats.md

```diff
@@ -1310,3 +1311,56 @@ The cells contain treatment response values that can be postive and negative rea
+
+### Generic Assay meta file
+The meta file will be similar to meta files of other genetic profiles, such as mRNA expression. For generic assay data `LIMIT-VALUE` is used as `datatype`. The `LIMIT-VALUE` is validated to contain any continuous number optionally prefixed with a '>' or '<' threshold symbol (e.g., '>8.00'). For generic assay meta files values for `stable_id` are not pre-registered in the cBioPortal implementation, but can be freely chosen by the user. Requirement is that each generic assay meta file is assigned an unique `stable_id` by the user. This requirement is validated by the data validation scripts. 
+
+Required fields: 
+```
+cancer_study_identifier: Same value as specified in meta file of the study
+genetic_alteration_type: GENERIC_ASSAY
+generic_assay_type: the actual data type
+datatype: LIMIT-VALUE
+stable_id: Any unique identifier using a combination of alphanumeric characters, _ and -
+profile_name: A name describing the analysis.
+profile_description: A description of the data processing done.
+data_filename: <name of generic assay data file>
+show_profile_in_analysis_tab: true
+generic_entity_meta_properties: The meta properties you want to have in your meta data, you need to specify them here, properties must seperated by column (e.g., 'name,description')
+```
+
+Example:
+```
+cancer_study_identifier: study_es_0
+genetic_alteration_type: GENERIC_ASSAY
+generic_assay_type: MUTATIONAL_SIGNATURE
+datatype: LIMIT-VALUE
+stable_id: mutational_signature
+profile_name: data of mutational signature
+profile_description: Description of data of mutational signature
+data_filename: data_mutational_signature.txt
+show_profile_in_analysis_tab: true
+generic_entity_meta_properties: name,description
+```
+
+#### Note on `generic_entity_meta_properties`
+All meta properties must be specified in `generic_entity_meta_properties`. Also, all meta properties specified in `generic_entity_meta_properties` are required to be shown in the data file as a same name column.
+
+#### Note on `Generic Assay` genetic_alteration_type and `LIMIT-VALUE` datatype
+all generic assay data is registered to be of the type of `genetic_alteration_type` and data type `LIMIT-VALUE`. This alteration and data type is intended to be used for any numerical data set with similar structure (entities measured in samples).
+
+### Generic Assay data file
+The data file will be a simple tab separated format, similar to the expression data file: each sample is a column, each generic assay is a row, each cell contains values for that generic_assay x sample combination.
```

Comment:
Can we follow the follow the format used to describe the expression file:

"For each generic entity (row) in the data file, the following columns are required..."

Reply:
@n1zea144 Could you have a look, is this better: https://github.com/cBioPortal/cbioportal/pull/6263/files#diff-fc80633ba06596c0353416ca1e72a3caR1356?